src/account-creator.ts

```diff
@@ -0,0 +1,81 @@
+import { Name } from '@greymass/eosio'
+import { ChainId } from 'eosio-signing-request'
+import { detectReturnPath } from '@greymass/return-path'
+
+import { AccountCreationOptions, AccountCreationResponse } from './types'
+
+export class AccountCreator {
+    private popupWindow?: Window
+    private scope?: Name
+    private supportedChains: ChainId[]
+    private whalesplainerUrl: string
```

Comment:
I might suggest this (and all instances of whalesplainer) use a more generic name, like `creationServiceUrl` - so our product name doesn't get baked in and potentially whalesplainer alternatives could be used?

Reply:
Good call! I made the change!